docs/content/guide/decorators.ngdoc

```diff
@@ -356,6 +356,7 @@ those conditions.
               // create and store the getters using $parse
               var getters = exps.map(function(el) {
                 if (el) return $parse(el);
+                return undefined;
```

Comment:
I think chagning the function body to `return el && $parse(el)` would be clearer for the reader.
Also, `el` seems quite unfitting. `exp` would be more intuitive (but this is unrelated to this PR :grin:)